CHANGES/1357.misc.rst

```diff
@@ -0,0 +1 @@
+new improved CallableMixin
```

Comment:
The description of the changes should answer the question "What was done?" and be understandable to users of the framework.
The given description doesn't answer this question and is not clear.